src/main/java/org/jabref/gui/externalfiles/FindFullTextAction.java

```diff
@@ -47,8 +55,9 @@ public FindFullTextAction(DialogService dialogService, BasePanel basePanel) {
     @Override
     public void action() {
         BackgroundTask.wrap(this::findFullTexts)
-                      .onSuccess(downloads -> SwingUtilities.invokeLater(() -> downloadFullTexts(downloads)))
-                      .executeWith(Globals.TASK_EXECUTOR);
+                .onSuccess(downloads -> SwingUtilities.invokeLater(() -> downloadFullTexts(downloads)))
```

Comment:
I think you no longer need the swing utitilies call here because you effectively removed the swing code which needed this.

Reply:
OK! Lambdas (and programming in general) are all new to me, I'm still getting used to them.  I have replaced this with a simple method reference to downLoadFullTexts. Thanks for the suggestion.  I was also able to remove the now unnecessary import of swing also.